src/lib.rs

```diff
@@ -116,6 +116,7 @@ mod token;
 pub use event::{
     PollOpt,
     EventSet,
+    IoEvent,
```

Comment:
rustc's privacy tracking is not so thorough - if something is declared with `pub` rustc considers it public and allows it in exported signatures, it need not be actually exported at the crate level